quirks/password-rules.json

```diff
@@ -431,6 +431,9 @@
     "santander.de": {
         "password-rules": "minlength: 8; maxlength: 12; required: lower, upper; required: digit; allowed: [-!#$%&'()*,.:;=?^{}];"
     },
+    "secure.orclinic.com": {
+        "password-rules": "minlength: 6; maxlength: 15; required: lower, digit; allowed: ascii-printable;"
```

Comment:
Also, the `required` rule as it is currently written:

`required: lower, digit;`

isn't sufficient for the rule described in the screenshot from the site.

The site requires at least one lowercase letter _and_ at least one digit. The rule above, however, only requires one lowercase letter _or_ one digit.

This will represent the site's rules accurately:

`required: lower; required: digit;`

requiring both a lowercase letter and a digit to be present in the password.

Reply:
Thanks, @Cldfire! I didn’t realize the distinction between writing the rule that way. I’ll get that fixed.

Reply:
```suggestion
        "password-rules": "minlength: 6; maxlength: 15; required: lower; required: digit; allowed: ascii-printable;"
```